File: CaveRunner/Assets/CaveRun3D/Scripts/LoadLevel.js
//This script just loads a level, set from the inspector
import UnityEngine.SceneManagement;
var LevelName:String = "game"; //The level's name

function Start()
{
  Debug.Log('Dynamic Level Loading - ' + LevelName);
  SceneManager.LoadScene(LevelName); //Load the level
}

File: CaveRunner/Assets/CaveRun3D/Scripts/LoadLevel.cs
using UnityEngine;
using UnityEngine.SceneManagement;

public sealed class LoadLevel : MonoBehaviour
{
    //This script just loads a level, set from the inspector
    string LevelName = "game"; //The level's name

    private void Start()
    {
        Debug.Log("Dynamic Level Loading - " + LevelName);
        SceneManager.LoadScene(LevelName); //Load the level
    }
}